electron/handlers/app.js

```diff
@@ -44,9 +44,14 @@ export function handleSecondInstance (
   args
 ) {
   if (mainWindow) {
-    mainWindow.show()
-
-    mainWindow.focus()
+    if (args && args.includes(
+      '--open-about-window'
+    )) {
+      aboutWindow.show()
+    } else {
+      mainWindow.show()
```

Comment:
Because mainWindow should have no behaviour when the action is to open the about window.